### PR TITLE
Feature 186: Example sentences for adjectives and verbs

### PR DESCRIPTION
**Pull request by phortx**
- Extracts a partial for the example sentences part of the show view.
- Adds the example sentences to the form and show views of verbs and adjectives too.

### CODE REVIEW

**Comment by wintermeyer** [comment]
Bei mir steht "This branch has conflicts that must be resolved".

**Comment by phortx** [comment]
Rebased :)

File: app/views/example_sentences/_list.html.haml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- if word.example_sentences.present?
+  = two_column_card t('words.show.example_sentences.title'), t('words.show.example_sentences.description') do
+    = box padding: false, class: 'striped' do
+      - word.example_sentences.each do |sentence|
+        .p-4
```
Comment (phylor) [inline app/views/example_sentences/_list.html.haml:5]:
```suggestion
        .p-4.flex.flex-wrap.justify-between
```

Comment (phylor) [reply app/views/example_sentences/_list.html.haml:5]: This might have been lost in the rebase?

**Review by phylor — APPROVED** [review]
Nice! :tada: 

In theory we could also add it for `FunctionWord`s, but not sure if there will be example sentences for those in practice.

File: app/views/example_sentences/_list.html.haml (hunk 1 above)
Comment (phortx) [reply app/views/example_sentences/_list.html.haml:5]: Wow, well spotted, Serge! Yes, that was los in the rebase conflict.

**Comment by phortx** [comment]
Thanks for the review! I'd suggest to add it for function words, when this is really required. :)

**Pull request merged**